test/blackbox-tests/test-cases/test-build-if/feature.t

```diff
@@ -26,7 +26,7 @@ We test the various combinations:
   >   else
   >     echo '  build was done: NO'
   >   fi
-  >   if echo $output|grep -q "alias runtest" ; then
```

Comment:
The way this test is setup is a little fragile. @emillon any suggestions for improving it?

Reply:
You can try printing something to `stderr` from `t.ml` to determine if the test did run.

Reply:
I've used an alert in t.ml to check for compilation and printed something to check for running.